### PR TITLE
feat(ISV-5313): set a name of index image sbom

### PR DESCRIPTION
**Pull request by Allda**
A name of index image SBOM is now set to image pullspec that includes registry, repository and image digest.

This feature is done to unify naming of the SBOMs across multiple content types.

JIRA: ISV-5313

### CODE REVIEW

**Comment by Allda** [comment]
@chmeliik @mkosiarc Can you guys please review or find somebody who could do that? Thanks

File: sbom-utility-scripts/scripts/index-image-sbom-script/index_image_sbom_script.py [hunk 1]
```diff
@@ -107,11 +119,11 @@ def create_sbom(
     image_index_digest: str,
     inspect_input: dict[str, Any],
 ) -> dict:
-    if inspect_input["mediaType"] != "application/vnd.oci.image.index.v1+json":
+    if inspect_input["mediaType"] not in INDEX_IMAGE_MANIFEST_MEDIA_TYPES:
```
Comment (chmeliik) [inline sbom-utility-scripts/scripts/index-image-sbom-script/index_image_sbom_script.py:122]: Could you extract the mediaType changes into a separate commit? They seem unrelated to the SBOM name changes

**Review by chmeliik — COMMENTED** [review]
LGTM, just a nitpick for unrelated changes in the same commit

**Review by chmeliik — APPROVED** [review]
